src/components/documentManager.ts

```diff
@@ -0,0 +1,533 @@
+import { BaseItemDto } from '../api/generated/models/base-item-dto';
+
+import { JellyfinApi } from './jellyfinApi';
+
+import { parseISO8601Date } from '../helpers';
+
+// Duration between each backdrop switch
+const backdropPeriodMs: number = 30000;
+
+// DocumentManager should never ha
```

Comment:
Ha what?

Reply:
Probably an unfinished justification for having it as a static class